www/js/diary/infinite_scroll_list.js

```diff
@@ -320,6 +320,12 @@ angular.module('emission.main.diary.infscrolllist',['ui-leaflet',
         $scope.data.displayTimelineEntries.push(place);
       }
     });
+
+    // if we are showing places and the last place does not have a display_end_time, 
+    // then we will use the display_start_time + 1 hour for display_end_time
+    if (!$scope.data.displayTimelineEntries[$scope.data.displayTimelineEntries.length-1].display_end_time && $scope.showPlaces) {
+      $scope.data.displayTimelineEntries[$scope.data.displayTimelineEntries.length-1].display_end_time = moment($scope.data.displayTimelineEntries[$scope.data.displayTimelineEntries.length-1].enter_fmt_time).parseZone().add(1, 'hour').format("h:mm A");
```

Comment:
If you see the discussion on https://github.com/e-mission/e-mission-docs/issues/871, I think that they expect it to be the end of the day.